app/src/main/java/org/fossasia/susi/ai/skills/skilldetails/adapters/recycleradapters/FeedbackAdapter.kt

```diff
@@ -44,28 +50,45 @@ class FeedbackAdapter(
         return 0
     }
 
+    private fun arrangeFeedbacks() {
+            arrangedFeedbackList.clear()
+            feedbackResponse.feedbackList.forEach { item ->
+                if (item.email == PrefManager.getStringSet(Constant.SAVED_EMAIL)?.iterator()?.next()) {
```

Comment:
`PrefManager.getStringSet(Constant.SAVED_EMAIL)?.iterator()?.next()` what?

Reply:
> `PrefManager.getStringSet(Constant.SAVED_EMAIL)?.iterator()?.next()` what?

This method has been used throughout the app while extracting email from the PrefManager. So, I used it else I would have simply used ```PrefManager.getString(Constant.SAVED_EMAIL,"")```

Reply:
Use yours

Reply:
Updated

Reply:
@iamareebjamal the app is crashing if I use my getString, but it is working if I use getStringSet. Probably that's the reason getStringSet has been used throughout the app.

Reply:
OK, revert but it is very stupid

Reply:
Already reverted in the latest update.